message_ix_models/model/material/util.py

```diff
@@ -272,8 +262,11 @@ def cost_fit(df) -> float:
 
 
 def update_macro_calib_file(scenario: message_ix.Scenario, fname: str) -> None:
-    """
-    Function to automate manual steps in MACRO calibration
+    """Function to automate manual steps in MACRO calibration
```

Comment:
```suggestion
    """Automates manual steps in MACRO calibration.
```

Typically, one would likely shorten the first sentence like this.

Also, would it be possible to get a test calling this function?